### PR TITLE
Packaging fixes

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

**Comment by jayvdb** [comment]
> storyhub/sdk/AutoUpdateThread.py | 87.5% <0%> (-12.5%)

A known bug, in our tests, our CI or in `coverage` or codecov?

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,4 @@
+include LICENSE
+include tox.ini
```
Comment (wilzbach) [inline MANIFEST.in]: Shouldn't we include the README here as well?

**Comment by wilzbach** [comment]
> A known bug, in our tests, our CI or in coverage or codecov?

Looks like a bug in the tests here.

File: MANIFEST.in (hunk 1 above)
Comment (jayvdb) [reply MANIFEST.in]: Not necessary.  modern setuptools automatically includes it.  Here is the top level of the sdist tarball

> LICENSE  MANIFEST.in  PKG-INFO  README.md  setup.cfg  setup.py  storyhub  story_hub.egg-info  tests  tox.ini

However if you want it to be explicitly added, if we are using old setuptools in our CD, I can do that.

**Pull request merged**